src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/DocumentType.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.entity;
+
+import javax.persistence.AttributeConverter;
+
+public enum DocumentType {
+
+    CHERISHED,
+    OTHER;
+
+    public static class Converter implements AttributeConverter<DocumentType, String> {
```

Comment:
I'd rather extract this converter out of DocumentType, so that the enum itself can stay unaware of those conversions.